projects/packages/forms/src/blocks/field-slider/index.js

```diff
@@ -28,6 +28,10 @@ const settings = {
 			type: 'number',
 			default: 0,
 		},
+		step: {
+			type: 'number',
+			default: 1,
```

Comment:
When working on number field, I was torn having this as string instead of number, and store value `any` as the default just like in the [native `step` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/range#step). It wouldn't then get stored in attributes because it was defined as default value, and only numbers would get stored.

Number works too and is clearer, just different tradeoffs.

Reply:
For now will keep this as is unless strong feeling otherwise. We can adjust later if needed.